src/wattsi.pas

```diff
@@ -447,10 +541,22 @@    TCrossReferences = record
             end;
             DFNEntry.SubDFNElement := Element;
          end;
+         SectionNumber := Default(Rope);
+         for CurrentHeadingRank := 2 to LastHeadingRank do
+         begin
+            if (CurrentHeadingRank > 2) then
+               SectionNumber.Append($002E);
+            SectionNumber.Append(IntToStr(CurrentSectionNumber[CurrentHeadingRank]));
+         end;
          Inc(DFNEntry.UsageCount);
          New(CrossRefListNode);
          CrossRefListNode^.Topic := CrossReferenceName;
          CrossRefListNode^.LastHeading := LastSeenHeadingID;
+         if ((SplitFilename = 'indices') or (SplitFilename = 'acknowledgements')) then
```

Comment:
Could this be solved without hardcoding these filenames? The headings have a `no-num` class...